certz/certz.proto

```diff
@@ -223,6 +288,36 @@ message RotateCertificateResponse {
 message FinalizeRequest {
 }
 
+// A request to add one or more SSL profiles.
+message AddProfileRequest {
+  // A list of IDs of SSL profiles to be created.
+  repeated string ssl_profile_ids = 1;
+}
+
+// A response to a request to add one or more SSL profiles.
+message AddProfileResponse {
+}
```

Comment:
What is the expected handling for the case where multiple IDs are provided here, and one of them fails to be added? 
For example, given ssl_profile_ids of  `[]string{ "foo", "bar", "baz" }`, imagine profile `foo` is created successfully but there is some error with creating profile `bar`.
We are limited here to only return either a single `AddProfileResponse` or an error. 

The error code itself wouldn't suffice to indicate which profiles were added successfully, so the client would have to do parsing on the error description to figure out which profiles were added and which caused errors. 

Two potentially cleaner ways to deal with this are:
1. make `AddProfileResponse` contain a map of the results of the request, similar to https://github.com/openconfig/gnoi/blob/79709cdf28e160f0732ff0086902c06c9c2d74f8/packet_link_qualification/packet_link_qualification.proto#L53
Something like:
```protobuf
// AddProfileResponse will return a map of the status of an AddProfileRequest.
// The map key is the SSL profile ID provided in the AddProfileRequest.
// Valid status responses are:
// OK: SSL profile had been added successfully.
// ALREADY_EXISTS: if the SSL profile ID is already present on the target.
// UNKNOWN: the target cannot add this profile for some unexpected reason.
message AddProfileResponse {
  map<string, google.rpc.Status> status = 2;
}
```

2. limit AddProfileRequest so that it only supports adding a single SSL profile, like:
```protobuf
// A request to add a new SSL profile.
message AddProfileRequest {
  // The ID of the SSL profile to be created.
  string ssl_profile_id = 1;
}
```

Reply:
The (silent) assumption is that the RPC is atomic. If one of the sub-operations fails then the whole RPC fails (with error message saying that a particular profile has not been created).

So, in my opinion in addition to the to options you have mentioned we have third one: improve the documentation.

I think option 2 works for me best.

Reply:
> (with error message saying that a particular profile has not been created).

Parsing the error description to find the cause of the RPC failure in this would create hassle for the client - unless the error description's format was standardized, the client would also have to perform different parsing for each vendor's error description.
 
I think it would be more robust to not need to parse the error description to know exactly which profile ID caused the RPC to fail. 

Option 2 also works best for me, just because it feels like the most simple option (one request for one ssl_profile_id = one response or error)

Reply:
I have removed the `repeated` keyword. Please taka a look.

Reply:
the changes look good to me - thanks Tomek!